tests/mpi/compute_index_owner_01.cc

```diff
@@ -20,6 +20,7 @@
 
 #include <deal.II/base/mpi.h>
 #include <deal.II/base/mpi_compute_index_owner_internal.h>
+#include <deal.II/base/mpi_consensus_algorithms.templates.h>
```

Comment:
@marcfehling Have you run this? I would say that the problem is:

```
    Utilities::MPI::ConsensusAlgorithms::Selector<
      std::vector<std::pair<types::global_dof_index, types::global_dof_index>>,
      std::vector<unsigned int>>
```

Reply:
You explicitly use non-vector variant here: https://github.com/dealii/dealii/blob/9f1ea188b76280fb18098a8bf8b5777a678a107f/tests/mpi/compute_index_owner_01.cc#L63-L66 Or am I mistaken?

Reply:
Wasn't this merely forgotten in #13722 (since this test does not run on any of the CI)?

Reply:
Exactly. See https://github.com/hyperdeal/hyperdeal/pull/99/files#diff-63bd370697e20555f6e5fe506d2f00a26d567e504330ec99e52cadfdb0256c20.

Reply:
Okay, I will change the test accordingly.